sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -443,6 +444,34 @@ public async Task AnalyzeHealthcareEntitiesPagination()
             Assert.AreEqual(2, pages[0].Count);
         }
 
+        [RecordedTest]
+        public async Task RecognizeHealthcareEntitiesWithFhir()
+        {
+            TextAnalyticsClient client = GetClient(useTokenCredential: true);
+
+            AnalyzeHealthcareEntitiesOptions options = new AnalyzeHealthcareEntitiesOptions()
+            {
+                FhirVersion = FhirVersion.Four01
```

Comment:
I'm guessing this is just how it generates?

Reply:
woah.

<img src="https://media1.giphy.com/media/fqbsVaM1ZMxm47vAgy/giphy.gif"/>

Reply:
Yes :( Issue https://github.com/Azure/azure-sdk-for-net/issues/28811 to make it better next release :)